<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPMain</class>
 <widget class="QMainWindow" name="LPMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Life Preserver</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lPreserve.qrc">
    <normaloff>:/images/tray-icon-idle.png</normaloff>:/images/tray-icon-idle.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="combo_pools">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="push_configure">
        <property name="statusTip">
         <string>Configure local data backup schedules</string>
        </property>
        <property name="text">
         <string>Configure</string>
        </property>
        <property name="icon">
         <iconset resource="lPreserve.qrc">
          <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_configBackups">
        <property name="statusTip">
         <string>Configure off-disk data backup solutions</string>
        </property>
        <property name="text">
         <string>Backups</string>
        </property>
        <property name="icon">
         <iconset resource="lPreserve.qrc">
          <normaloff>:/images/drive-removable-media.png</normaloff>:/images/drive-removable-media.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_status">
       <attribute name="icon">
        <iconset resource="lPreserve.qrc">
         <normaloff>:/images/drive-removable-media.png</normaloff>:/images/drive-removable-media.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>System State:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_status">
            <property name="text">
             <string notr="true">GOOD</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Number of Disks:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_numdisks">
            <property name="text">
             <string notr="true">1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Latest Snapshot:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_latestsnapshot">
            <property name="text">
             <string notr="true">snapshot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_replication">
          <property name="statusTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(10,200,10,100); border-radius: 5px</string>
          </property>
          <property name="text">
           <string notr="true">Replication Status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_mirror">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(10,100,200,100); border-radius: 5px</string>
          </property>
          <property name="text">
           <string notr="true">Mirror Status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_errors">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(230,10,10,100); border-radius: 5px</string>
          </property>
          <property name="text">
           <string notr="true">System Errors</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_restore">
       <attribute name="icon">
        <iconset resource="lPreserve.qrc">
         <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Restore Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="combo_datasets"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="push_prevsnap">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lPreserve.qrc">
              <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QSlider" name="slider_snapshots">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_snapshot">
              <property name="styleSheet">
               <string notr="true">border: 1px solid grey; border-radius: 5px</string>
              </property>
              <property name="text">
               <string notr="true">Selected Snapshot</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="push_nextsnap">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="lPreserve.qrc">
              <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="push_restore">
            <property name="text">
             <string>Restore</string>
            </property>
            <property name="icon">
             <iconset resource="lPreserve.qrc">
              <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
       <zorder>line_2</zorder>
       <zorder>treeView</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionManage_Pool"/>
    <addaction name="actionUnmanage_Pool"/>
    <addaction name="separator"/>
    <addaction name="action_SaveKeyToUSB"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Window"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDisks">
    <property name="title">
     <string>Disks</string>
    </property>
    <addaction name="actionAdd_Disk"/>
    <addaction name="actionRemove_Disk"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Disk_Offline"/>
    <addaction name="separator"/>
    <addaction name="action_startScrub"/>
   </widget>
   <widget class="QMenu" name="menuSnapshots">
    <property name="title">
     <string>Snapshots</string>
    </property>
    <addaction name="action_newSnapshot"/>
    <addaction name="action_rmSnapshot"/>
   </widget>
   <widget class="QMenu" name="menuClassic_Backups">
    <property name="title">
     <string>Classic Backups</string>
    </property>
    <addaction name="actionCompress_Home_Dir"/>
    <addaction name="actionExtract_Home_Dir"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuClassic_Backups"/>
   <addaction name="menuDisks"/>
   <addaction name="menuSnapshots"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose_Window">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Close Window</string>
   </property>
  </action>
  <action name="actionAdvanced">
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionBasic">
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionManage_Pool">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Manage Pool</string>
   </property>
  </action>
  <action name="actionUnmanage_Pool">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Unmanage Pool</string>
   </property>
  </action>
  <action name="actionRecover_File">
   <property name="text">
    <string>Recover File</string>
   </property>
  </action>
  <action name="actionRestore_Dataset">
   <property name="text">
    <string>Restore Dataset</string>
   </property>
  </action>
  <action name="actionSave_Key_to_USB">
   <property name="text">
    <string>Save Key to USB</string>
   </property>
  </action>
  <action name="actionAdd_Disk">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add Disk</string>
   </property>
  </action>
  <action name="actionRemove_Disk">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove Disk</string>
   </property>
  </action>
  <action name="actionSet_Disk_Offline">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/network-wired.png</normaloff>:/images/network-wired.png</iconset>
   </property>
   <property name="text">
    <string>Set Disk Offline</string>
   </property>
  </action>
  <action name="actionNew_Snapshot">
   <property name="text">
    <string>New Snapshot</string>
   </property>
  </action>
  <action name="action_newSnapshot">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/camera_add.png</normaloff>:/images/camera_add.png</iconset>
   </property>
   <property name="text">
    <string>New Snapshot</string>
   </property>
  </action>
  <action name="action_rmSnapshot">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete Snapshot</string>
   </property>
  </action>
  <action name="action_SaveKeyToUSB">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/key.png</normaloff>:/images/key.png</iconset>
   </property>
   <property name="text">
    <string>Save Key to USB</string>
   </property>
  </action>
  <action name="action_startScrub">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/system-run.png</normaloff>:/images/system-run.png</iconset>
   </property>
   <property name="text">
    <string>Start Scrub</string>
   </property>
   <property name="statusTip">
    <string>Check system data integrity</string>
   </property>
  </action>
  <action name="actionClassic_Backup">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/drive-removable-media.png</normaloff>:/images/drive-removable-media.png</iconset>
   </property>
   <property name="text">
    <string>Classic Backup</string>
   </property>
  </action>
  <action name="actionCompress_Home_Dir">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/box_add.png</normaloff>:/images/box_add.png</iconset>
   </property>
   <property name="text">
    <string>Compress Home Dir</string>
   </property>
  </action>
  <action name="actionExtract_Home_Dir">
   <property name="icon">
    <iconset resource="lPreserve.qrc">
     <normaloff>:/images/box_extract.png</normaloff>:/images/box_extract.png</iconset>
   </property>
   <property name="text">
    <string>Extract Home Dir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="lPreserve.qrc"/>
 </resources>
 <connections/>
</ui>
